internal/logger/config.go

```diff
@@ -0,0 +1,27 @@
+package logger
+
+import (
+	"os"
+	"time"
+
+	"github.com/dicedb/dice/config"
+	"github.com/rs/zerolog"
+)
+
+var l zerolog.Logger
```

Comment:
IMO, we should move away from the practise of using global variables towards creating an instance using a New<Resource>() method:
There are a few disadvantages I see with using global variables:
1. Less Testable as the state is global and writing unit tests become difficult.
2. Lack of Flexibility as it becomes difficult to adjust logger configuration.
3. Global variables couple the code to a specific library creating a tight coupling.

Reply:
Thanks @soumya-codes I was actually waiting for your review to use either New() or go with the global, had both things in my mind. The only disadvantage I see with New() provinding instance everytime, we need to get it everytime and use it in each function. With global variable that's not an issue. Your thoughts?